inductiva/_cli/cmd_tasks/list.py

```diff
@@ -79,14 +92,24 @@ def register(parser):
     group.add_argument("-n",
                        "--last-n",
                        type=int,
-                       help="List last N tasks. Default: 5.")
-    group.add_argument("-id",
-                       "--task-id",
+                       default=10,
+                       help="List last N tasks. Default: 10.")
+    group.add_argument("-a",
+                       "--all",
+                       dest="all_tasks",
+                       action="store_true",
+                       help="List all tasks.")
+    group.add_argument("-i",
```

Comment:
maybe we can accept multiple IDs:

```suggestion
    group.add_argument("-i", ..., nargs='+', ...)
```